doc/Support/FAQ.md

```diff
@@ -28,6 +28,7 @@ source: Support/FAQ.md
  - [What is the Difference between Disable Device and Ignore a Device?](#faq28)
  - [Why can't Normal and Global View users see Oxidized?](#faq29)
  - [What is the Demo User for?](#faq30)
+ - [Why does modifying 'Default Alert Template' fail?](#faq31)
```

Comment:
Actually, the listing on the on alert templates page is static. So it'll appear whether the record is in the db or not.